datafusion/common/src/table_reference.rs

```diff
@@ -139,7 +142,7 @@ impl TableReference {
 
     /// Convenience method for creating a [`TableReference::Partial`].
     ///
-    /// As described on [`TableReference`] this does *NO* parsing at all.
+    /// As described on [`TableReference`] this does *NO* normalization at all.
```

Comment:
nit: technically the doc on `TableReference` doesn't make reference to `partial` or `full` functions as it only refers to `bare`

Reply:
That is a good call -- I have clarified in a1d5965fff9767f6d44088e547275e69edaa81bc. Thank you @Jefffrey